plugin-build/src/test/kotlin/io/sentry/android/gradle/util/SentryPluginUtilsTest.kt

```diff
@@ -36,36 +34,6 @@ class SentryPluginUtilsTest(private val agpVersion: SemVer) {
     assertEquals("", "".capitalizeUS())
   }
 
-  @Test
```

Comment:
I had to remove all these tests as they are now running with AGP 8 which is not compatible with proguard.
IMO, we should drop all these proguard tests completely since it isn’t really supported with AGP 8+.